tests/system/data/test_system_async.py

```diff
@@ -14,13 +14,16 @@
 
 import pytest
```

Comment:
We added [CrossSync](go/cross_sync_design) to this repo to generate the sync code from the async base. Since you're touching the async files, you''ll have to run `nox -s generate_sync` before merging (and a unit test will fail if the sync client isn't up to date, so you'll be reminded)

In more complex code, you sometimes need to add CrossSync annotations to the code to guide the generator, but you shouldn't have to do anything for the changes here, outside what you already have. But let me know if you run into issues with it

Reply:
Done, thanks

Reply:
Sorry I realized there was still something failing from the generated code. Regenerated and fixed now